.github/workflows/build-apk.yml

```diff
@@ -48,13 +49,43 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x ./gradlew
 
-      # Build the APK
+      # Decode the Keystore from secret
+      - name: Decode Keystore
+        env:
+          ANDROID_KEYSTORE_BASE64: ${{ secrets.ANDROID_KEYSTORE_BASE64 }}
+        run: |
+          mkdir -p ~/.android
+          echo "$ANDROID_KEYSTORE_BASE64" | base64 --decode > ~/.android/debug.keystore
+
+        # Generate SHA1 fingerprint from keystore
+      - name: Generate SHA1
```

Comment:
I believe this steps shows the content of the keystone, it doesn't generate the SHA1. This is already stored as a key in the key store, and applied when calling apksigner. This is useful for debugging, and making sure everything is in order, but for a proper release not necessary. 
If left, rename the step (if my interpretation is correct)

Reply:
yes your interpretation is correct I forgot to change it, I will leave as it is but modify the name